include/poddlthread.h

```diff
@@ -12,10 +12,10 @@
 
 namespace podboat {
 
-class poddlthread {
+class PoddlThread {
```

Comment:
Should this be named `PodDlThread`?

Reply:
That makes a lot of sense. I'll change all of those to PodDlThread